### ISSUE TITLE
Does Mock Firebase Storage supports 'delete()' functionality?

### ISSUE THREAD

**Issue by faheem4797** [atn832/firebase_storage_mocks#38]
For reference:

await mockFirebaseStorage.refFromURL(imageUrl).delete();

final newUrl = await mockFirebaseStorage.refFromURL(imageUrl).getDownloadURL(); 

newUrl should throw an exception but it gets the actual url meaning the file isn't deleted

**Comment by atn832** [comment]
Yep, looks like a bug. If you have time, would you mind proposing a PR? I'm not sure when I could fix this.

**Comment by atn832** [comment]
Just published @faheem4797 's fix to https://pub.dev/packages/firebase_storage_mocks/changelog#0801.
